src/styles/main.scss

```diff
@@ -1,3 +1,199 @@
+$blue: #00acdc;
+$dark-blue: #060b35;
+$gray: #616070;
+$white: #fff;
+$border-color: #f3f3f3;
+$font-family: "Roboto", sans-serif;
+
+@import "header";
+
+html {
+  color: $dark-blue;
+  font-size: 8px;
+  font-family: $font-family;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
```

Comment:
devide main.scss by files, styles related to nav move to nav.scss, star => star.scss, cart => cart.scss etc